.github/workflows/playwright.yml

```
@@ -42,7 +42,7 @@ jobs:
         run: npx playwright install --with-deps
       - name: Run Playwright tests
         run: make test
-      - uses: actions/upload-artifact@v2
+      - uses: actions/upload-artifact@v4.4.0
```

```suggestion
      - uses: actions/upload-artifact@v4
```

Doesn't really matter, but we don't pin many of the actions in the other repos. And the official GH actions are pretty well versioned, so if there are any non-breaking improvements on 4.X we get them automatically here.